### ISSUE TITLE
Remove certain specs, refer to Waku

### ISSUE THREAD

**Issue by decanus** [status-im/specs#73]
So there are now multiple sections which we have specified both in this repository and in https://github.com/vacp2p/specs, I propose that we remove the duplicates from this repository and refer back to vac. This will start transitioning this spec to more of implementation decisions rather than a full technical spec as we had previously planned.

cc @corpetty @oskarth @adambabik

**Comment by adambabik** [comment]
I think we can do that when we actually switch the app to use Waku. It will take some time until this happens.

**Comment by oskarth** [comment]
We could to this. However, it might make sense to switch a bit later, considering:

1. Status spec needs to get to a state to reflect v1 (audit, core release). This is the effort @corpetty is/will be leading here https://github.com/status-im/specs/issues/65 It'd be good to have more clarity on when and how that is going to be achieved.

2. Waku is still moving and hasn't been integrated end to end yet. It might make sense for this to happen before moving over and deduplicating.

We can talk about it briefly in the call today.

PS Added to Waku board

**Comment by corpetty** [comment]
I agree with @oskarth and @adambabik.  It's fine to have some redundancy for now in lieu of getting the specs to V1 as fast as possible.

**Comment by decanus** [comment]
@corpetty yeah we came to that conclusion on the call, we are leaving this issue open however to not forget about it.

**Comment by oskarth** [comment]
See https://github.com/vacp2p/pm/issues/7
